### PR TITLE
One click build doc

### PR DESCRIPTION
**Pull request by NeoZhangJianyu**
1. correct the requirements.txt
2. add script to build online doc by one click.

### CODE REVIEW

**Review by chensuyue — APPROVED** [review]

**Review by yinghu5 — APPROVED** [review]
let's try

**Review by mkbhanda — APPROVED** [review]
Minor comments 
line 36: echo "repo ${REPO} exists, skipping cloning"
line 57: echo "CP html to ${PUBLISHDIR}" .. better to write "Copied html content to $(PUBLISHDIR}"

LGTM

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+make html
+echo "Build online doc done!"
+
+echo "update github.io"
+
+RELEASE_FOLDER=../opea-project.github.io
+BUILDDIR=_build
+PUBLISHDIR=${RELEASE_FOLDER}/latest
+
+cp -r ${BUILDDIR}/html/*  ${PUBLISHDIR}
```
Comment (dbkinder) [inline scripts/build.sh:52]: @NeoZhangJianyu @mkbhanda @yinghu5 @chensuyue You left out a step that's in the "make publish" steps that handles the case when a file was deleted from the previously published documents.  You should remove everything from the PUBLISHDIR folder before copying al the files that were just built:

```
rm -rf $(PUBLISHDIR)/*
```

File: scripts/setup_env.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+sudo apt install git doxygen graphviz -y
```
Comment (dbkinder) [inline scripts/setup_env.sh:3]: @NeoZhangJianyu @mkbhanda @yinghu5 @chensuyue @intelkevinputnam OPEA doesn't use doxygen?

Comment (mkbhanda) [reply scripts/setup_env.sh:3]: Thank you @dbkinder for continuing to be the doc guardian angle for OPEA!! 
https://doxygen.nl/index.html Doxygen "automates the generation of documentation from source code comments, parsing information about classes, functions, and variables to produce output in formats like HTML and PDF" but I see no signs of its use in our various OPEA subprojects. Just see references to it in opea-projects/docs in scripts/setup_env.sh and scripts/show-versions.py. For that matter I do not see graphviz either. Are these being included in anticipation of use @NeoZhangJianyu ?

**Comment by NeoZhangJianyu** [comment]
> Minor comments line 36: echo "repo ${REPO} exists, skipping cloning" line 57: echo "CP html to P U B L I S H D I R " . . b e t t e r t o w r i t e " C o p i e d h t m l c o n t e n t t o (PUBLISHDIR}"
> 
> LGTM

Yes, I will update as your comment later.

Thank you!

File: scripts/build.sh (hunk 1 above)
Comment (NeoZhangJianyu) [reply scripts/build.sh:52]: Yes, I will update as your comment later.

Thank you!

**Pull request merged**